### ISSUE TITLE
Doc request: Add group manipulation example to micro examples

### ISSUE THREAD

**Issue by KennethNielsen** [SoCo/SoCo#757]
_Making this issue for a friend_

The examples page in our docs (http://docs.python-soco.com/en/latest/examples.html) consist of micro examples on how to perform some small task. It would be great if they would be extended with examples of group manipulation. More specifically:

* How to make a group (possibly showing the changes in topology)
* How to play something on the group (keeping track of which is the master)
* How the break the group up.
* Brief mention of "partymode"

Implementing this feature request is a suitable task to start with for a new contributor, as it requires only knowing how to do group manipulation in SoCo.

**Comment by jnth** [comment]
Hi,
I wrote a new section in the documentation about the group manipulation.
What do you think of this ?

**Comment by DPH** [comment]
Great addition - thanks. 
I would just add after "Only the coordinator of a group can control the group." 
```
eg what's playing. Render commands such as setting volume,  loudness, mute etc are still sent to each player. 
```
or similar wording to make it clear
Cheers David

**Comment by jnth** [comment]
@DPH I add a few lines to clarify the coordinator role.

**Comment by DPH** [comment]
Thanks for clarifying Jonathan,
I'm not sure if what you say is correct though unless there have been
changes.

My understanding is:
Volume,  Loudness,  mute, Night mode etc. are controlled on each individual
player not by the group.
Queue, Play_uri, Play, Next, Pause etc. are controlled by the coordinator
for all players in the group.

Basically the members of the group point their source to the coordinator
for what to play.
The volume etc. for individual members is controlled for each player
separately.

Hope that makes sense,
Cheers David

**Comment by jnth** [comment]
Sorry, my mistake !
I fixed the section.

Thanks David for the advice.

**Comment by DPH** [comment]
Great - thanks for your contribution.
These micro examples are very important to help new users understand SoCo.
Great work.

Cheers David

**Comment by pwt** [comment]
Resolved in #807. Thanks to @jnth, and to @DPH for the reviews.

**Comment by KennethNielsen** [comment]
Thanks @jnth and @DPH
